tests/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -105,8 +105,8 @@ local utils = import "templates/utils.libsonnet";
     command: utils.scriptCommand(
       |||
         %(common)s --model_name_or_path roberta-large \
-        --per_gpu_train_batch_size 16 \
-        --per_gpu_eval_batch_size 16
+        --per_device_train_batch_size 16 \
+        --per_device_eval_batch_size 16
```

Comment:
For these tests, you have metrics assertion for `eval_mnli/acc` and `eval_mnli-mm/acc`
if you think those keys will be pretty stable, I'd recommend adding:
`required_metrics: ['eval_mnli/acc', 'eval_mnli-mm/acc']`
to each of the remaining tests in this file

Reply:
Done.